src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../../../public/variables";
+
+@mixin background($color,$shape) {
+  background-color: #{$color};
+  clip-path: $shape;
+  padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
+}
```

Comment:
If we split them, do we even need the mixin? since each variable addresses a different CSS prop. I'm okay with it as is, but if we want to split it, i'd remove the mixin all together.